codalab/model/tables.py

```diff
@@ -93,6 +93,73 @@
     Column('parent_path', Text, nullable=False),
 )
 
+# Storage location for bundles.
+bundle_store = Table(
+    'bundle_store',
+    db_metadata,
+    Column(
+        'id',
+        BigInteger().with_variant(Integer, "sqlite"),
+        primary_key=True,
+        nullable=False,
+        autoincrement=True,
+    ),
+    Column('uuid', String(63), nullable=False),
+
+    # Usually root. For BYO bundle stores (not yet supported),
+    # it is the user who created the bundle store.
+    Column('owner_id', String(255), nullable=True),
+
+    # Name used to refer to this bundle store from the CLI.
+    Column('name', String(255), nullable=True),
+
+    # Storage type. This is usually redundant with information already in the url column, but set for efficiency reasons. When updating this column, sync it with codalab.common.StorageType.
+    Column(
+        'storage_type',
+        Enum(StorageType.DISK_STORAGE.value, StorageType.AZURE_BLOB_STORAGE.value),
+        nullable=True,
+    ),
+
+    # The format through which the bundle is stored.
+    Column(
+        'storage_format',
+        Enum(StorageFormat.UNCOMPRESSED.value, StorageFormat.COMPRESSED_V1.value)
+    ),
+
+    # URL that uniquely identifies the bundle store.
+    Column('url', String(255), nullable=True),
+    
+    # Authentication / password to access the bundle store.
+    Column('authentication', String(255), nullable=True),
+
+    # Name of environment variable through which the authentication variable from above can be passed to codalab-service to access the bundle store.
+    Column('authentication_env', String(255), nullable=True),
+
+    UniqueConstraint('uuid', 'owner_id', name='uix_1'),
```

Comment:
I think we discussed that we would make it unique on uuid only (instead of `(uuid, owner_id)`)

Reply:
done